12-watchtower-API.md

```diff
@@ -270,73 +317,58 @@ Sample code (python) for the client to prepare the `encrypted_blob`:
 	    encrypted_blob = hexlify(encrypted_blob).decode()
 	
 	    return encrypted_blob
+	    
+## Signed Receipt
 
-## Signed Receipt Fields
+### Sanity checks
 
-### Sanity check
-
-The reading node (WatchTower) must perform a sanity check on the job request before sending a signed receipt to the writer (customer). The sanity check is implementation-specific, but we recommend that the WatchTower checks the satoshi per byte (```transaction_fee/transaction_size```) and that the dispute delta satisifies a minimum (i.e. sufficient time to complete the job). 
+The server (WatchTower) must perform some sanity checks on the job request before sending a signed receipt to the client. The sanity checks are implementation-specific, but we recommend that the WatchTower checks the satoshi per byte (`transaction_fee/transaction_size`) and that the dispute delta satisfies a minimum (i.e. sufficient time to complete the job). 
```

Comment:
you already covered a lot of the checks previously? is there a way to combine both sections?

Reply:
This was part of your writing. I'm OK with removing it.

Reply:
ok